node/src/accountant/big_int_db_processor.rs

```diff
@@ -1376,94 +1437,77 @@ mod tests {
                 Ok((high_bytes, low_bytes))
             })
             .unwrap();
-        UpdateWithOverflowSummary {
-            balance_change_decomposed: BigIntDivider::deconstruct(balance_change_signed),
-            final_read_high_bytes: final_high_bytes,
-            final_read_low_bytes: final_low_bytes,
-            math_operation_expected_result: BigIntDivider::reconstitute(
-                final_high_bytes,
-                final_low_bytes,
-            ),
-        }
+        (final_high_bytes, final_low_bytes)
     }
 
     #[test]
     fn update_with_overflow_for_addition() {
-        let initial_high_bytes = 4555;
-        let initial_low_bytes = i64::MAX - 55;
-        //signed just to make the test more unconditional
-        let balance_change_signed = i64::MAX as i128 * 4; //36893488147419103228;
+        let big_initial = i64::MAX as i128 * 3;
+        let big_addend = i64::MAX as i128 + 454;
+        let big_sum = big_initial + big_addend;
 
-        let result = update_with_overflow_shared_test_body(
+        let (final_high_bytes, final_low_bytes) = update_with_overflow_shared_test_body(
             "update_with_overflow_for_addition",
-            (initial_high_bytes, initial_low_bytes),
-            balance_change_signed,
-            Addition,
+            big_initial,
+            Addition("balance", big_addend as u128),
         );
 
         assert_eq!(
-            result,
-            UpdateWithOverflowSummary {
-                balance_change_decomposed: (3, 9223372036854775804),
-                final_read_high_bytes: 4555 + 3 + SINGLE_UNIT_FROM_OVERFLOW,
-                final_read_low_bytes: 9223372036854775804 - 55 - SINGLE_UNIT_FROM_OVERFLOW,
-                math_operation_expected_result: BigIntDivider::reconstitute(4555, i64::MAX - 55)
-                    + i64::MAX as i128 * 4
-            }
+            BigIntDivider::deconstruct(big_initial),
+            (2, 9223372036854775805)
         );
+        assert_eq!(BigIntDivider::deconstruct(big_addend), (1, 453));
+        let result = BigIntDivider::reconstitute(final_high_bytes, final_low_bytes);
+        assert_eq!(result, big_sum)
     }
 
     #[test]
     fn update_with_overflow_for_subtraction_from_positive_num() {
-        let initial_high_bytes = 4555;
-        let initial_low_bytes = 55;
-        //signed just to make the test more unconditional
-        let balance_change_signed = -41;
+        let big_initial = i64::MAX as i128 * 2;
+        let big_subtract = i64::MAX as i128 + 120;
+        let big_sum = big_initial - big_subtract;
```

Comment:
Maybe nobody cares about this level of nomenclature, but in the equation

A - B = C

A is the minuend, B is the subtrahend, and C is the difference.